src/main/java/seedu/address/model/ReadOnlyInterviewSchedule.java

```diff
@@ -3,13 +3,14 @@
 import javafx.collections.ObservableList;
 import seedu.address.model.interview.Interview;
 
+//@@author lzan98
 /**
  * Unmodifiable view of an interview list
  */
 public interface ReadOnlyInterviewSchedule {
     /**
-     * Returns an unmodifiable view of the candidates list.
-     * This list will not contain any duplicate candidates.
+     * Returns an unmodifiable view of the interview list.
+     * This list will not contain any duplicate interviews.
```

Comment:
Thanks for making the change in the JavaDocs😄

Reply:
no probs just spotted it while going thru :)